packages/cli/src/util/client.ts

```diff
@@ -1,7 +1,11 @@
 import { MedplumClient, MedplumClientOptions } from '@medplum/core';
 import { FileSystemStorage } from '../storage';
+import { Profile } from '../utils';
 
-export async function createMedplumClient(options: MedplumClientOptions): Promise<MedplumClient> {
+export async function createMedplumClient(
+  options: MedplumClientOptions,
+  profileObject?: Profile
+): Promise<MedplumClient> {
   const profile = options.profile ?? 'default';
```

Comment:
This seems high risk of tying us in knots, because we have 2 separate arguments that specify profile, and it's not clear which one we're using.

I think we should probably be clear here whether we're using the `profile` argument or `options.profile`.

My recommendation would be something like this:

```ts

if (!profile) {
    profile = loadProfile(options.profile ?? 'default'); // or getProfile or readProfile or whatever
}
```